### PR TITLE
Changes Line 33 and line 81

### PR DESCRIPTION
**Pull request by YogeshBarania**
Added section separately due to requirement instead of adding to benefits. Also, pls add pointing URL from Admin requirement section to line 33 in the end so that users can navigate to it directly

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -30,10 +30,7 @@ Centralized Deployment is the recommended and most feature-rich way for most cus
 
 Centralized Deployment provides the following benefits:
 
-- A Global admin or an exchange admin can assign an add-in directly to a user, to multiple users via a group, or to everyone in the organization.
-- Exchange admin can deploy addin only if App Registration property is set to true in Azure Active Directory admin center as shown in below image.
-
-  ![image](../../media/AAD.png)
+- An admin can deploy and assign an add-in directly to a user, to multiple users via a group, or to everyone in the organization (see Admin requirement section for info)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
- An admin can deploy and assign an add-in directly to a user, to multiple users via a group, or to everyone in the organization (see Admin requirement section for information).
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 2]
```diff
@@ -81,6 +78,15 @@ Microsoft Exchange stores the add-in manifests within your organization's tenant
 
 Check with your organization's Exchange admin to find out which configuration is in use. OAuth connectivity per user can be verified by using the [Test-OAuthConnectivity](/powershell/module/exchange/test-oauthconnectivity) PowerShell cmdlet.
 
+### Admin requirements
+
+In order to deploy an add-in via Centralized Deployement, you need to be either a Global admin or an Exchange admin in the organization.
+
+> [!NOTE]
+>Exchange admin can deploy add-in only if **App Registrations** property is set to true in Azure Active Directory admin center as shown below:
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
> An Exchange admin can deploy an add-in only if the **App Registrations** property is set to true in Azure Active Directory admin center as shown in the following image:
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 3]
```diff
@@ -81,6 +78,15 @@ Microsoft Exchange stores the add-in manifests within your organization's tenant
...
+>
+>![image](https://user-images.githubusercontent.com/89943918/144516704-8874a10d-b540-41f3-ae9d-c07a8d7e143f.png)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
> ![image](https://user-images.githubusercontent.com/89943918/144516704-8874a10d-b540-41f3-ae9d-c07a8d7e143f.png)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @YogeshBarania, kindly commit the suggested changes. Many thanks.

**Comment by YogeshBarania** [comment]
Changes done as per suggestion, pls approve and publish this doc., thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**